build.xml

```diff
@@ -1580,56 +1580,17 @@
     </target>
 
 
-    <target name="alltest-coveragereport" depends="logalltest" description="Generate Test Coverage Report from logalltest">
-    <!-- Generate Code Coverage report
-    See: http://www.eclemma.org/jacoco/trunk/doc/ant.html -->
-       <jacoco:report>
-          <executiondata>
-              <file file="${jacocoexec}" />
-          </executiondata>
+    <target name="alltest-coveragereport" depends="logalltest, coveragereport" description="Generate Test Coverage Report from logalltest" />
 
-          <structure name="AntTestReporting">
-              <classfiles>
-                 <fileset refid="jacocofileset"/>
-              </classfiles>
-          </structure>
-
-          <html destdir="${coveragetarget}" />
-          <csv destfile="${coveragetarget}/report.csv"/>
-          <xml destfile="${coveragetarget}/report.xml"/>
-       </jacoco:report>
-    </target>
+    <target name="alltest-coveragecheck" depends="logalltest, coveragecheck" description="Test Coverage check from logalltest" />
 
-    <target name="alltest-coveragecheck" depends="logalltest" description="Test Coverage check from logalltest">
-    <!-- Generate Code Coverage report
-    See: http://www.eclemma.org/jacoco/trunk/doc/ant.html -->
-       <jacoco:report>
-          <executiondata>
-              <file file="${jacocoexec}" />
-          </executiondata>
+    <target name="ci-test-travis-coveragecheck" depends="ci-test-travis, coveragecheck" description="Test Coverage check from logalltest for Appveyor" />
 
-          <structure name="AntTestReporting">
-              <classfiles>
-                 <fileset refid="jacocofileset"/>
-              </classfiles>
-          </structure>
+    <target name="ci-test-appveyor-coveragecheck" depends="ci-test-appveyor, coveragecheck" description="Test Coverage check from logalltest for Travis" />
 
-          <check>
```

Comment:
Please restore the coverage check targets.  These are used to verify the coverage targets are met without generating the reports.

In particular, the appveyor version is used during Appveyor runs to verify we are meeting the coverage requirements.  This helps us spot issuing tests in packagetests for new classes.

Reply:
No targets were removed. They were all made to depend on a test target and either the `coveragereport` or `coveragecheck` target.